docs/overview/index.md

```diff
@@ -40,9 +50,9 @@ The `ZIO` data type is the only effect type in ZIO. However, there are a family
 
 **Tips For Getting Started With Type Aliases**
 
-If you are new to functional effects, we recommend starting with the `Task` type, which has a single type parameter and corresponds most closely to the `Future` data type built into Scala's standard library.
+If you are new to functional effects, we recommend starting with the `Task` type, which has a single type parameter and corresponds most closely to the `Future` data types built into the Scala and Java standard libraries.
 
-If you are using _Cats Effect_ libraries, you may find the `RIO` type useful, since it allows you to thread environments through third-party libraries and your application.
+If you are using _Cats Effect_ libraries, you may find the `RIO` type useful, since it allows you to thread cobntext through third-party libraries.
```

Comment:
```suggestion
If you are using _Cats Effect_ libraries, you may find the `RIO` type useful, since it allows you to thread context through third-party libraries.
```